.claude/skills/prd-create/SKILL.v1-careful.md

```
@@ -0,0 +1,207 @@
+- Created PRD for [brief feature description]
+- Defined [X] major milestones
+- Documented problem, solution, and success criteria
+- Added to ROADMAP.md ([timeframe] section)
+- Ready for implementation"
+
+# Pull latest and push to main
+git pull --rebase origin main && git push origin main
+```
+
+**Confirmation Message:**
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)